css/style.css

```diff
@@ -0,0 +1,359 @@
+/******************************************
+/* LAYOUT
+/*******************************************/
+#container {
+  width: 900px;
+  margin: 0 auto;
+  border: 1px solid black;
+}
+
+header {
+  height: 8em;
+  border: 1px solid black;
+  background-color: black;
+  border-bottom: 4px solid #F9F625;
+}
+
+header h1 {
+  color: #F9F625;
+  font-weight: bolder;
+  font-size: 2.4em;
+  font-family: serif;
+  margin-top: 0.6%;
+  margin-left: 2%;
+}
+
+#header-nav {
+  margin-left: 14%;
+  margin-top: 1%;
+  margin-bottom: 2%;
+}
+
+a.header-links {
+  color: white;
+  margin-right: 4%;
+}
+
+a.big {
+  font-family: impact;
+  font-size: 1.7em;
+}
+
+a.small {
+  font-family: serif;
+  font-size: 1.4em;
+}
+
+#small-match-p {
+  display: none;
+}
+
+#below-header-message {
+  height: 6em;
+  width: 100%;
+  background-color: rgba(128, 128, 128, 0.47);
+  text-align: center;
+}
+
+#below-header-message p {
+  color: black;
+  font-size: 2em;
+  font-style: italic;
+  font-family: serif;
+  font-weight: bolder;
+  padding-top: 2%;
+}
+
+#taste {
+  text-decoration: line-through;
+  color: rgba(128, 128, 128, 0.66);
+}
+
+#are {
+  border-bottom: 2px solid #F9F625; 
+  padding-bottom: 0.1%;
+}
+
+#image-container {
+/*  z-index: -1;*/
+  background: url(../images/coworking.jpg) no-repeat center top rgb(134,141,157);
+  height: 33em;
+  background-size: 100% 100%;
+}
+
+#meet-your-match {
+  width: 25%;
+  background-color: rgba(255, 255, 255, 0.72);
+  border: 1px solid rgba(0, 0, 0, 0.22);
+  position: relative;
+  top: 3em;
+  padding: 1%;
+  text-align: left;
+  color: black;
+  left: 69%;
+}
+
+#meet-your-match h4 {
+  font-size: 2em;
+  font-family: serif;
+  font-weight: bolder;
+  padding-top: 3%;
+  padding-left: 2%;
+  margin-bottom: 5%;
+}
+
+#meet-your-match p {
+  font-size: 1.3em;
+  font-family: serif;
+  margin: 3%;
+}
+
+#meet-your-match button {
+  height: 2.3em;
+  width: 93%;
+  background-color: #F9F625;
+  font-weight: bold;
+  font-family: impact;
+  font-size: 1rem;
+  border-radius: 5px;
+  border-color: transparent;
+  margin-top: 10px;
+  color: black;
+  margin-left: 3%;
+  margin-top: 5%;
+  margin-bottom: 3%;
+}
+
+#general-content {
+  height: 19em;
+  text-align: left; 
+}
+
+.general-content-titles {
+  font-size: 2.1em;
+  font-weight: bold;
+  font-family: initial;
+  margin-bottom: 5%;
+  margin-left: 13%;
+  margin-top: 2%;
+}
+
+.general-content-boxes {
+  width: 33.3%;
+  padding: 1%;
+  float: left;
+}
+
+.general-content-boxes p {
+  margin-left: 6%;
+  margin-right: 10%;
+  font-size: 1.2em;
+}
+
+.general-content-buttons {
+  height: 2.4em;
+  width: 70%;
+  background-color: rgba(128, 128, 128, 0.44);
+  font-weight: lighter;
+  font-family: impact;
+  font-size: 1.4em;
+  border-radius: 5px;
+  border-color: transparent;
+  margin-top: 10px;
+  color: black;
+  margin-left: 10%;
+  margin-top: 5%; 
+}
+
+#create-profile {
+  border-right: 1px solid gray;
+}
+
+#find-a-dev {
+  border-right: 1px solid gray;
```

Comment:
If I replace it with border then it will apply the border to the whole div, including top, bottom and left. To try to mimic the design I added the border to the right only on purpose. Please let me know if I should still do this or leave it like that. Thanks.